README.md

```diff
@@ -1,27 +1,148 @@
-This repository contains Giraffe, a React-based visualization library used to implement the [InfluxDB 2.0](https://github.com/influxdata/influxdb/) UI.
+# Giraffe
 
-**This library is currently in a pre-alpha state.**
-Docs will follow as soon as the API has a chance to stabilize.
+A React-based visualization library powering the data visualizations in [InfluxDB 2.0](https://github.com/influxdata/influxdb/) UI.
+
+**This library is currently in pre-beta**
 
 ## 🦒 Features
 
 There exist plenty of terrific visualization libraries in the JavaScript ecosystem.
 Giraffe aims to distinguish itself with several features:
 
+- Supports [Flux](https://www.influxdata.com/products/flux)
+- Easy reactivity and extensibility via React
+- Support for mapping groupings of columns to a single visual aesthetic
 - A high-level [_Grammar of Graphics_](http://vita.had.co.nz/papers/layered-grammar.pdf) style API that can specify a wide variety of visualizations with a few simple concepts
 - A [columnar](https://observablehq.com/@mbostock/manipulating-flat-arrays) interface for input data that enables efficient interop with Web Workers and [Apache Arrow](https://arrow.apache.org/)
-- Easy reactivity and extensibility via React
 - Self-contained configs in the style of [Vega-Lite](https://vega.github.io/vega-lite/)
-- Support for mapping groupings of columns to a single visual aesthetic
 
-## Development
+## Getting Started [](#getting-started)
+
+#### Installation
+
+Install [Giraffe](https://www.npmjs.com/package/@influxdata/giraffe) with your package manager
+
+`yarn add @influxdata/giraffe` or `npm install @influxdata/giraffe`
+
+#### Example
+
+1. In your React code, import the `Plot` component and the `newTable` utility function
+
+  <pre>
+  import {Plot, newTable} from '@influxdata/giraffe'
+  </pre>
+
+2. Build the config object.  
+   a. **Required properties**:
+
+   - `table` is data built using the newTable utilty function (_also_ built from Flux results, _see_ [Flux example](./README.md#example-using-flux))
+   - `layers` is an array of objects that describe how to render the data.
+
```

Comment:
There is a plug for the full configuration guide right after the next property. I think it's better there, after introducing both concepts, rather than leading the reader astray immediately with a link begging to be clicked.